vignettes/migration_from_the_old_crmPack.Rmd

```diff
@@ -46,10 +51,10 @@ class_changes <- tribble(
   "DALogisticLogNormal", "", "C_par", "c_par", FALSE, "",
   "DALogisticLogNormal", "", "conditionalPEM", "cond_pem", FALSE, "",
   "TITELogisticLogNormal", "", "weightMethod", "weight_method", FALSE, "",
-  "OneParLogNormalPrior", "", "skeletonFun", "skel_fun", FALSE, "",
+  "OneParLogNormalPrior", "", "", "skel_fun", FALSE, "",
```

Comment:
are these other changes here intended?

Reply:
Yes, please see my comment below. `OneParLogNormalPrior` is new, e.g. not part of 1.0.3. However we changed the name during development, but that is nothing we need to track (in my opinion).

Reply:
correct, agree